apps/neo4j/5.25.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   neo4j:
-    image: neo4j:5.25.1
+    image: neo4j:5.26.0
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
This patch introduces a small update to the Neo4j service configuration by changing its version from `5.25.1` to `5.26.0`. There are no obvious errors or optimization issues with this change; it is an update to keep the image up-to-date.